src/sharedServices/zendesk/updateZendeskTicket.ts

```diff
@@ -55,11 +55,7 @@ export const updateZendeskTicketById = async (
     }
   }
   try {
-    const data = await makeHttpsRequest(options, postData)
-    logger.info(
-      interpolateTemplate('zendeskSuccessful', loggingCopy),
-      JSON.stringify(data)
-    )
+    await makeHttpsRequest(options, postData)
```

Comment:
Probably still nice to know the update was successful. Maybe use ticket id instead of all data?

Reply:
As part of the work to log out success of all external calls, this has been done now. The ZendeskId is always in the context, so it's there by default in the new message we added.